.devcontainer/devcontainer.json

```diff
@@ -34,10 +34,12 @@
         "files.trimTrailingWhitespace": true,
         "terminal.integrated.profiles.linux": {
           "zsh": {
+            "env": {
+              "PATH": "/home/vscode/.local/bin/"
+            },
             "path": "/usr/bin/zsh"
           }
         },
-        "terminal.integrated.defaultProfile.linux": "zsh",
```

Comment:
This looks wrong, why delete this line ?

Reply:
It does not seem to be needed.
If I leave it in, the profile does not seem to append the PATH variable.

Reply:
I've added the default profile back in however pointing it to bash instead.
It seem like zsh is not used in the container and running `echo $SHELL` shows that it is using `bash`

```sh
vscode ➜ /workspaces/core (fix-devcontainer-missing_hass) $ zsh -c 'echo $SHELL'
/bin/bash
```